### ISSUE TITLE
Post-event content?

### ISSUE THREAD

**Issue by unnawut** [ThaiProgrammer/tech-events-calendar#70]
For events that are already on the calendar, is it okay or recommended to update them with links to the content generated from the event?

E.g. links to the recorded livestreams, code repos, photo galleries, blog posts, etc.

This would make the calendar not useful just for future events but as an archive/reference for past events as well.

**Comment by dtinth** [comment]
Yes please! This is a great suggestion and a great way to make this project more useful. To do this, we need to figure out how to put these information in the markdown file.

Proposal: the `resources` key inside the front matter:

```yaml
resources:
  - type: video
    title: Recorded sessions
    url: https://www.youtube.com/playlist?list=PL37ZVnwpeshFqN5dcZ704lxI3F5iHDYkl
```

Type may be:

- `video`
- `code`
- `photo`
- `writeup`
- `slides`

Another question: What if there are community create resource about the content? e.g. someone (not the event organizer) wrote a blog post. How should we display it?
 One suggestion is to have another key, `community_resources` which is the same as `resources` but for content created by community.

**Comment by unnawut** [comment]
Sounds great. I believe `lib/parseMarkdown.js` and `website/src/components/EventInfo.vue` are the only 2 files that need to be changed? If so I can try give it a go if you guys give the green light on the proposal.

**Comment by unnawut** [comment]
Ah sorry I forgot to add the mock visual with the previous comment ...

![screen shot 2561-02-26 at 23 24 25](https://user-images.githubusercontent.com/921194/36685262-5ba76da8-1b55-11e8-960f-e66a05232fdc.png)

**Comment by dtinth** [comment]
Closed by #72, thanks @unnawut!

(I added several more videos in #77 #78 #79.)
